dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -44,32 +47,32 @@ public class ObjectCreationNode extends Node {
      *
      * @param tree the NewClassTree
      * @param enclosingExpr the enclosing expression Node if it exists, or null
-     * @param constructor the constructor node
+     * @param identifier the identifier node
      * @param arguments the passed arguments
      * @param classbody the ClassDeclarationNode
      */
     public ObjectCreationNode(
             NewClassTree tree,
             @Nullable Node enclosingExpr,
-            Node constructor,
+            Node identifier,
             List<Node> arguments,
             @Nullable ClassDeclarationNode classbody) {
         super(TreeUtils.typeOf(tree));
         this.tree = tree;
         this.enclosingExpression = enclosingExpr;
-        this.constructor = constructor;
+        this.identifier = identifier;
         this.arguments = arguments;
         this.classbody = classbody;
     }
 
     /**
-     * Returns the constructor node
+     * Returns the identifier node.
```

Comment:
Add the same explanation as on the field here - it's even more visible and important here.

Reply:
Thanks! Done in my local.